Libraries/Azure.psm1

```diff
@@ -346,12 +346,12 @@ Function Create-AllResourceGroupDeployments($SetupTypeData, $TestCaseData, $Dist
                                 -StorageAccountName $used_SC
 
                         $DeploymentStartTime = (Get-Date)
-                        $CreateRGDeployments = Create-ResourceGroupDeployment -RGName $groupName -location $location -TemplateFile $azureDeployJSONFilePath `
```

Comment:
why are you removing location?

Reply:
Because the way it's used in the create deployment function does not make sense maybe a stronger check would be to get resource group and check it exists at this point but the location check I'm certain is pointless here.
The location is checked before we hit this point. Anyway the resource group creation is dependent on location, not the resource group deployment.